src/pages/starter-kit/checkout/tax-use-cases.md

```diff
@@ -204,3 +204,35 @@ Here's an example payload showing how the custom attributes from tax classes app
   }
 }
 ```
+## Propagate serialized custom attributes
+
+### Tax class custom attributes available in Quote
+
+The `oope-tax-management` module introduces support for assigning serialized custom attributes to tax classes. These attributes are then automatically associated with shopping cart data during cart creation and product addition. This allows for tax-related metadata to be included early in the checkout process and carried forward into subsequent operations.
+
+### How it works
```

Comment:
I'd remove this section title as it works without it and for aesthetic purpose of index appearing in the page (as it's too long now).